actionmailbox/lib/action_mailbox/routing.rb

```diff
@@ -17,6 +17,10 @@ def routing(routes)
       def route(inbound_email)
         router.route(inbound_email)
       end
+
+      def mailbox_for(inbound_email)
+        router.route(inbound_email)
```

Comment:
Yeah, I struggled with a good name here, but `ApplicationMailbox.mailbox_for(inbound_email)` seemed to scan alright. We want something that _isn't_ a command verb, since the point of this method is to extract out the side-effect free part of inbound email routing. Personally, I would describe this part as `route`ing and the existing `route` method as `receive`ing, but I know that change would be confusing (and I'm not sure if `route` is currently considered a public method).

Other ideas:

* `routing_target`
* `destination_mailbox`
* `match_to_mailbox` (the existing private method name)
* `matching_mailbox`

Any preferences or guidelines for a better name?

Reply:
Sorry, I’m saying this method needs to call `mailbox_for`, not `route`.

Reply:
Yup. 🤦‍♂ Updated. Thank you.